OpenRA.Mods.Common/Activities/Move/Follow.cs

```diff
@@ -47,5 +49,14 @@ public override Activity Tick(Actor self)
 
 			return ActivityUtils.SequenceActivities(path, this);
 		}
+
+		// Follow, is as in follow and attack (chase) so it is actually an attack activity.
+		// (What came to my mind first was follow and guard activity)
```

Comment:
This is used for attacking *and* guarding.  The guard case should return yellow.